onscale_client/client.py

```diff
@@ -2515,7 +2527,7 @@ def _validate_simulation(self, input_file: str, sim_materials: dict) -> dict:
         sims = load_sims(module)
         for sim in sims:
             validation_visitor = ValidationVisitor(material_map_path=material_map_path)
-            validation_visitor.analyze(sim)
+            # validation_visitor.analyze(sim)
```

Comment:
The `_validate_simulation()` call is commented out as well, see https://github.com/OnScale/OnScale-Cloud-Client/pull/14/files#diff-9887582e634f20d5aec1bde454175ccea0a6447ba382f3fdffeb313718631d42R1621

I cannot remove altogether all the validation methods right now because they are needed to build the simulation metadata. The minute we don't need those, we can get rid of all these validation methods.

Reply:
ok